### ISSUE TITLE
Review test testUtils.createSinonSandbox usage in tests

### ISSUE THREAD

**Issue by jodator** [ckeditor/ckeditor5#1144]
During reivew of stubbing `env.isEdge` in Image @oskarwrobel & me noticed strange behavior when stubbing using sinon sandbox. I've forgot about restoring stubbed property but sinon didn't complain about double stubbing of `env.isEdge`.

It looks like we might be using `testUtils.createSinonSandbox();` in tests files.

Check the output of logging `afterEach()` added by image tests:

![selection_010](https://user-images.githubusercontent.com/247363/42452838-6e28b600-838b-11e8-8d09-122d904a07b8.png)

To test this I've added logging to the  `testUtils.createSinonSandbox();`
```js
	createSinonSandbox(a) {
		before( () => {
			console.log(a)
			utils.sinon = sinon.sandbox.create();
		} );

		afterEach( () => {
			console.log(a, 'restore')
			utils.sinon.restore();
		} );
	},
```

and used it some tests:

```js
//...
testUtils.createSinonSandbox( 'ImageStyleEditing' );

describe( 'ImageStyleEditing', () => {
	let editor, model, document, viewDocument;
/...
```

---
tl;dr: we should put `testUtils.createSinonSandbox();` in the first `describe()` of a test file - not before because it will add `before()` & `afterEach()` callbacks to all tests.

**Comment by jodator** [comment]
Right now we have ~67 `testUtils.createSinonSandbox()` used before `describe` in all packages.

**Comment by oskarwrobel** [comment]
Luckily it does not affects tests quality. 
> I've forgot about restoring stubbed property but sinon didn't complain about double stubbing of env.isEdge

We figured out that sinon does not complain about double stub of properties. It does it only for functions. So it not related.

Anyway, sandbox restores too many times.

**Comment by jodator** [comment]
Done in all repositories which test were using `testUtils.createSinonSandbox()`.
